client/src/components/JurisdictionAdmin/generateSheets.test.tsx

```diff
@@ -276,4 +309,238 @@ describe('generateSheets', () => {
       )
     })
   })
+
+  describe('downloadBatchTallySheets', () => {
+    // Snapshots generated by a Mac Ubuntu VM can differ ever so slightly from those generated
+    // by CircleCI in a way that isn't visually perceptible, so we tolerate a 0.1% diff
+    // TODO: Get to the root cause of the diffs
```

Comment:
Will open a GitHub issue for this if I can't figure it out shortly!

Reply:
One idea is to selectively remove pieces of content from the PDF to see if there's a specific part that causes the diff. But also the tiny diff tolerance seems fine

Reply:
One idea is to selectively remove pieces of content from the PDF to see if there's a specific part that causes the diff. But also the tiny diff tolerance seems fine

Reply:
Definitely - I have a few courses of action here that I'm going to try before I fully shift back to L&A

Reply:
https://github.com/votingworks/arlo/issues/1490